python/pynessie/commands/contents.py

```diff
@@ -170,18 +182,13 @@ def _edit_contents(key: str, body: str) -> Optional[str]:
 
 
 def _get_contents(
-    nessie: NessieClient, ref: str, delete: bool = False, use_stdin: bool = False, expect_same_contents: bool = False, *keys: str
+    nessie: NessieClient, ref: str, keys: List[str], delete: bool = False, use_stdin: bool = False, expect_same_contents: bool = False
 ) -> List[Operation]:
     contents_altered: List[Operation] = list()
+    contents_dict = nessie.get_values(ref, keys)
```

Comment:
We don't intend to check for `NessieNotFoundException` in case reference is not found here?

Reply:
That's already handled. It prints something like
```
Requested URL: http://localhost:19120/api/v1/trees/contents?ref=foop
Server status: 404
Error code: REFERENCE_NOT_FOUND
Server message: Named reference 'foop' not found
Server traceback: None
```